include/hpp/core/problem.hh

```diff
@@ -296,6 +300,8 @@ namespace hpp {
     protected:
       /// \copydoc Problem::create(DevicePtr_t);
       Problem (DevicePtr_t robot);
+      /// \copydoc Problem::create(DevicePtr_t);
+      Problem (const Problem& other);
```

Comment:
Is this different from the default copy constructor ?
If no, I suggest to change this to
```suggestion
      Problem (const Problem& other) = default;
```
and then to remove the implementation in the definition file.

Reply:
Thank you for the suggestion. I will update and merge.